modular_skyrat/modules/clock_cult/code/structures/gear_base.dm

```diff
@@ -0,0 +1,123 @@
+/obj/structure/destructible/clockwork/gear_base
+	name = "gear base"
+	desc = "A large cog lying on the floor at feet level."
+	icon_state = "gear_base"
+	clockwork_desc = "A large cog lying on the floor at feet level."
+	anchored = FALSE
+	break_message = span_warning("Oh, that broke. I guess you could report it to the coders, or just you know ignore this message and get on with killing those god damn heretics coming to break the Ark.")
+	/// What's appeneded to the structure when unanchored
+	var/unwrenched_suffix = "_unwrenched"
+	/// Lazylist of nearby transmission signals
+	var/list/transmission_sigils
+	/// Makes sure the depowered proc is only called when its depowered and not while its depowered
+	var/depowered = FALSE
+	/// Minimum power to work
+	var/minimum_power = 0
+
+
+// UNCOMMENT TRANSMISSION SIGIL CODE WHEN IT IS ADDED
+
+/obj/structure/destructible/clockwork/gear_base/Initialize(mapload)
+	. = ..()
+	update_icon_state()
+	LAZYINITLIST(transmission_sigils)
+//	for(var/obj/structure/destructible/clockwork/sigil/transmission/trans_sigil in range(src, SIGIL_TRANSMISSION_RANGE))
+//		link_to_sigil(trans_sigil)
+
+/obj/structure/destructible/clockwork/gear_base/Destroy()
+//	for(var/obj/structure/destructible/clockwork/sigil/transmission/trans_sigil as anything in transmission_sigils)
+//		trans_sigil.linked_structures -= src
+	return ..()
+
+
+/obj/structure/destructible/clockwork/gear_base/wrench_act(mob/living/user, obj/item/tool)
+	if(!(IS_CLOCK(user)))
+		return
+
+	to_chat(user, span_notice("You begin to [anchored ? "unwrench" : "wrench"] [src]."))
+	if(!tool.use_tool(src, user, 2 SECONDS, volume = 50))
+		return
+
+	to_chat(user, span_notice("You successfully [anchored ? "unwrench" : "wrench"] [src]."))
```

Comment:
I feel like the first one could be a balloon_alert and the second, a visual_message(), so that others around would see it too, like when you wrench/unwrench machinery.